$MirOS: ports/archivers/zip/patches/patch-unix_Makefile,v 1.3 2007/02/23 11:04:31 bsiegert Exp $
--- unix/Makefile.orig	Sat Apr 22 23:51:55 2006
+++ unix/Makefile	Fri Feb 23 13:31:19 2007
@@ -17,10 +17,9 @@ SHELL = /bin/sh
 LN = ln -s
 
 # (to use the Gnu compiler, change cc to gcc in CC)
-CC = cc
 BIND = $(CC)
 AS = $(CC) -c
-CPP = /lib/cpp
+CPP = $(CC) -E
 EXE =
 
 # probably can change this to 'install' if you have it
@@ -47,7 +46,7 @@ VERSION = Version 2.32
 #   LFLAGS1   flags after output file spec, before obj file list
 #   LFLAGS2   flags after obj file list (libraries, etc)
 CFLAGS_NOOPT = -I. -DUNIX $(LOCAL_ZIP)
-CFLAGS = -O2 $(CFLAGS_NOOPT)
+CFLAGS += $(CFLAGS_NOOPT)
 LFLAGS1 =
 LFLAGS2 = -s
 
@@ -128,18 +127,16 @@ $(ZIPMANUAL): man/zip.1
 # install
 install:        $(ZIPS)
 	-$(INSTALL_D) $(BINDIR)
-	$(INSTALL_PROGRAM) $(ZIPS) $(BINDIR)
-	-cd $(BINDIR); $(CHMOD) $(BINFLAGS) $(ZIPS)
+	$(BSD_INSTALL_PROGRAM) $(ZIPS) $(BINDIR)
 	-$(INSTALL_D) $(MANDIR)
-	$(INSTALL) man/zip.1 $(MANDIR)/zip.$(manext)
-	$(CHMOD) $(MANFLAGS) $(MANDIR)/zip.$(manext)
+	$(BSD_INSTALL_MAN) man/zip.1 $(MANDIR)/zip.$(manext)
 
 uninstall:
 	-cd $(BINDIR); rm -f $(ZIPS)
 	-cd $(MANDIR); rm -f zip.$(manext)
 
 flags:  unix/configure
-	sh unix/configure "${CC}" "${CFLAGS_NOOPT}"
+	sh unix/configure ${CC:Q} ${CFLAGS:M*:Q}
 
 # These symbols, when #defined using -D have these effects on compilation:
 # ZMEM                  - includes C language versions of memset(), memcpy(),
